test/unit/css.js

```diff
@@ -664,6 +654,190 @@ QUnit.test( "show() after hide() should always set display to initial value (#14
 	assert.equal( div.css( "display" ), "list-item", "should get last set display value" );
 } );
 
+QUnit.test( "show/hide 3.0, default display", function( assert ) {
```

Comment:
I would move elements to their own test and then divide it again on `show` and `hide` tests too.

This sounds bad, since this is a lot of work and we don't do stuff like that in jQuery suite, but i would really want our tests to became more focused and clean.

Like moving this in their own `QUnit.module` (not into to separate file but into the module), and start using `beforeEach` here, like
```js
QUnit.module('css:show/hide default display', {
	beforeEach: function() {
		this.$div = jQuery( "<div></div>" ).appendTo( "#qunit-fixture" );
		this.$li = jQuery( "<ul><li></li></ul>" ).appendTo( "#qunit-fixture" ).find( "li" );
		// ... and so on
	}
});

QUnit.test( "hide sequence of `div` (block) element", function( assert ) {
	assert.expect( 3 );
	for ( i = 0; i < 3; i++ ) {
		this.$div.show().hide();
		assert.equal( this.$div.css('display'), 'none' );
	}
});

QUnit.test( "show sequence of `div` (block) element", function( assert ) {
	assert.expect( 3 );
	for ( i = 1; i < 4; i++ ) {
		this.$div.hide().show();
		assert.equal( this.$div.css('display'), 'block' );
	}
});
```
Here, we are still writing tests like we (well, maybe not "we" but @dmethvin :-) did literally ten years ago, 
whereas now js testing has evolved so much.

Yeah, this kinda a different topic, wouldn't hurt to start somewhere, hurt if we would start to do it right though. 

All these thoughts shouldn't block this pr though.